src/boot/pcrphase.c

```diff
@@ -0,0 +1,262 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
+
+#include <sd-messages.h>
+
+#include "efivars.h"
+#include "main-func.h"
+#include "openssl-util.h"
+#include "parse-util.h"
+#include "pretty-print.h"
+#include "tpm-pcr.h"
+#include "tpm2-util.h"
+
+static char *arg_tpm2_device = NULL;
+static char **arg_banks = NULL;
+
+STATIC_DESTRUCTOR_REGISTER(arg_banks, strv_freep);
+STATIC_DESTRUCTOR_REGISTER(arg_tpm2_device, freep);
+
+static int help(int argc, char *argv[], void *userdata) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd-pcrphase", "1", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+               "\n%5$sMeasure boot phase into TPM2 PCR 11.%6$s\n"
```

Comment:
shouldn't the PCR be selectable with a switch (defaulting to 11)?

Reply:
so far all our PCR assignments are hardcoded, and defined as macros in `src/fundamental/tpm-pcr.h`. We use exactly those definitions here, too.

I really want to push people to use a common vocabulary of TPM assignments, much like the kernel also doesn't make it configurable to use any other PCR than 9 for the initrds. 

Should this ever become an issue we can certainly revisit this: making something that is currently hardcoded configurable is always easy.